hack/build-and-push.sh

```diff
@@ -431,6 +434,34 @@ attach_migration_file() {
     return 0
 }
 
+# Find previous bundle that has a migration and output its digest.
+find_previous_migration_bundle_digest() {
+    local -r task_name=${1:?Missing task name}
+    local -r task_version=${2:?Missing task version}
+    local -r repo=${TEST_REPO_NAME:-task-${task_name}}
+
+    local -r url_list_recent_tags="https://quay.io/api/v1/repository/${QUAY_NAMESPACE}/${repo}/tag/?onlyActiveTags=true&limit=30"
```

Comment:
Is there a specific reason for `limit=30`? Shall we just use Quay's maximum, i.e. `limit=100`?

Reply:
For the link, the first tag is enough to build the link. I selected 30 for my local testing by using a single test image repo. Updated it to 100 so that it can cover more cases for convenience.